pkg/server/server.go

```diff
@@ -1058,6 +1076,18 @@ func (s *ImmuServer) UseDatabase(ctx context.Context, req *schema.Database) (*sc
 		return nil, err
 	}
 
+	if auth.GetAuthTypeFromContext(ctx) == auth.SESSION_AUTH {
```

Comment:
I personally prefer this `auth.SESSION_AUTH` over camel case constants, but I think camel case is the standard in Golang ...